packages/ui/src/components/Typography/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const TYPOGRAPHY_CLASS_PREFIX = 'condo-typography' as const
+export const TEXT_SIZES = ['lg', 'md', 'sm'] as const
+export const TEXT_TYPES = ['secondary', 'inverted', 'danger', 'warning', 'info', 'success'] as const
```

Comment:
What the reason to use `export … as const`, since you already declared a const?

Reply:
It's the way to tell typescript that's it is "true" const, so it can safely generate types on top of it

Reply:
I don't see any errors from TypeScript

Tested in following way:

module 1:

```ts
export const TEST_ARRAY = [1, 2]
```

module 2:

```ts
TEST_ARRAY.push(3)
```

Reply:
`as const` makes the constant read-only. 
Typescript will say that we cannot change it (including the elements in the array)

Reply:
Yep... @Alllex202 is right...And we can use read-only constants in type definitions